docs/source/faq.md

```diff
@@ -115,6 +115,33 @@ It allows customization of the various parameters. An environment variable
 has precedence over the value defined in `ucx.conf`.
 The file can be created using `ucx_info -Cf`.
 
+#### Build user application with UCX
+
+UCX supports metainformation subsystem based on `pkg-config` tool. To add UCX support
+into application, output from `pkg-config` tool may be used in Makefile:
```

Comment:
UCX supports metainformation subsystem based on `pkg-config` tool. In order to build the
application with UCX development libraries.
For example, this is how pkg-config can be incorporated in a Makefile-based build: